src/olympia/accounts/serializers.py

```diff
@@ -14,11 +14,11 @@ class Meta:
         model = UserProfile
         fields = (
             'username', 'display_name', 'email',
-            'bio', 'deleted', 'display_collections',
+            'biography', 'deleted', 'display_collections',
```

Comment:
Docs need to updated (this is a backwards-incompatible change, but hopefully nobody uses that API...)